mtp_api/apps/transaction/api/cashbook/serializers.py

```diff
@@ -44,3 +44,14 @@ class Meta:
             'refunded',
             'refunded_at',
         )
+
+
+class LockedTransactionSerializer(TransactionSerializer):
+    locked = serializers.BooleanField(read_only=True)
+    locked_at = serializers.DateTimeField(read_only=True)
+
+    class Meta(TransactionSerializer.Meta):
+        fields = TransactionSerializer.Meta.fields + (
+            'locked',
+            'locked_at',
+        )
```

Comment:
Is there a particular reason to only showing these fields for the locked view? Might be that the overall list could show that status too…